ignite/pkg/cosmosclient/txservice.go

```diff
@@ -29,9 +28,13 @@ func (s TxService) Broadcast() (Response, error) {
 	accountName := s.clientContext.GetFromName()
 	accountAddress := s.clientContext.GetFromAddress()
 
-	if err := s.client.prepareBroadcast(context.Background(), accountName, []sdktypes.Msg{}); err != nil {
-		return Response{}, err
-	}
+	// TODO uncomment after https://github.com/tendermint/spn/issues/363
```

Comment:
Why is it an issue?

In `cosmosclient`, we use `c.lockBech32Prefix()`. that should set the correct bech32 prefix for the chain. `spn` in this case

Reply:
I have no idea, this TODO was moved from a removed method `prepareBroadcast` which is no longer useful (see https://github.com/ignite/cli/pull/2781#discussion_r955991287). Tell me if I can remove it.

Reply:
Putting back the commented code should work

Reply:
Done here 77a2a6d0c2f89f42bd3c7e7dba958077d757e54a

The node txs integration are still green, so sounds good for me. Note that I'm currently writing more tests for cosmosclient, including the tx part.

Reply:
Yes working with `spn` as well